{
  "Header": {
    "Original": "CVE-2018-20169 (An issue was discovered in the Linux kernel before 4.19.9. The USB sub ...)",
    "Line": 279499,
    "ID": "CVE-2018-20169",
    "Description": "(An issue was discovered in the Linux kernel before 4.19.9. The USB sub ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-1771-1 DLA-1731-1}",
      "Line": 279500,
      "Type": "xref",
      "Bugs": [
        "DLA-1771-1",
        "DLA-1731-1"
      ]
    },
    {
      "Original": "- linux 4.19.9-1",
      "Line": 279501,
      "Type": "package",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "4.19.9-1"
    },
    {
      "Original": "[stretch] - linux 4.9.161-1",
      "Line": 279502,
      "Type": "package",
      "Release": "stretch",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "4.9.161-1"
    },
    {
      "Original": "NOTE: https://git.kernel.org/linus/704620afc70cf47abb9d6a1a57f3825d2bca49cf",
      "Line": 279503,
      "Type": "NOTE",
      "Description": "https://git.kernel.org/linus/704620afc70cf47abb9d6a1a57f3825d2bca49cf"
    }
  ]
}